sources/configure.ml

```diff
@@ -116,6 +118,7 @@ let () =
   let () = Format.fprintf fmt "# Generated by configure@." in
   let () = Format.fprintf fmt "prefix=%s@." !prefix in
   let () = Format.fprintf fmt "libdir=%s@." !libdir in
+  let () = Format.fprintf fmt "profile=%s@." (if !static then "static-release" else "release") in
```

Comment:
With these modifications, the only two profile used will be `release`and `static-release`, ignoring the `dev` profile entirely, which is not optimal I think.